conanfile.py

```diff
@@ -40,8 +40,8 @@ def build_requirements(self):
         self.build_requires("gtest/1.14.0")
 
     def requirements(self):
-        self.requires("homestore/[>=6.2,      include_prerelease=True]@oss/master")
-        self.requires("sisl/[>=12.1, include_prerelease=True]@oss/master")
+        self.requires("homestore/[~6.2.0, include_prerelease=True]@oss/master")
```

Comment:
This should be ~6.2, right? So that HO can still pick any patch changes automatcially from HS? 
Or is `~6.2.0` also do the work?